QUICK_START_GUIDE.md

```diff
@@ -103,6 +104,9 @@
                            Load only weights from previously saved checkpoint
      --save-model-to SAVE_MODEL_TO
                            Location where trained model will be stored.
+     --enable-hpo          Excute hyper parameters optimization (HPO) before training.
+     --hpo-time-ratio HPO_TIME_RATIO
+                           Expected ratio of total running time when executing HPO.
```

Comment:
which ratio ?

Reply:
Description for hpo-time-ratio is updated.